temp/modemmanager: Fix sending SMS with UTF-16 characters

Upstream: no

Patch recommended by Aleksander Morgado in upstream issue, until a proper
fix is developed:
https://gitlab.freedesktop.org/mobile-broadband/ModemManager/-/issues/274#note_671679

diff --git a/src/mm-charsets.c b/src/mm-charsets.c
index 19d1874c4..67b2103cd 100644
--- a/src/mm-charsets.c
+++ b/src/mm-charsets.c
@@ -43,7 +43,7 @@ static const CharsetEntry charset_map[] = {
     { "PCCP437", "CP437",  "CP437",     "CP437//TRANSLIT",     MM_MODEM_CHARSET_PCCP437 },
     { "PCDN",    "CP850",  "CP850",     "CP850//TRANSLIT",     MM_MODEM_CHARSET_PCDN    },
     { "HEX",     NULL,     NULL,        NULL,                  MM_MODEM_CHARSET_HEX     },
-    { "UTF-16",  "UTF16",  "UTF-16BE",  "UTF-16BE//TRANSLIT",  MM_MODEM_CHARSET_UTF16   },
+    { "UTF-16",  "UTF16",  "UTF-16BE",  "UTF-16BE",  MM_MODEM_CHARSET_UTF16   },
 };
